RELEASES.md

```diff
@@ -1,3 +1,11 @@
+# 0.7.0
+### Change
+- Update `dusk-bls12_381` to `0.5.0`.
+- Downgrade `rand` to `0.7`.
+- Downgrade `rand_core` to `0.5`.
+- Downgrade `rand_xorshift` to `0.2`.
+- Closes #56, implements `Serializable` for all structures.
```

Comment:
I would avoid to mentioned issues like that, they don't get linked to the actual issue. Also, they should just mentioned at the end. I noticed it was doing in the past but it's not correct.

I'll be sure to formalize the changelog format once we finish with priorities, but the important thing here is that you're doing a breaking change in the API – `from_bytes` DO NOT return a `Option` / `CtOption` anymore. That's the important thing to highlight.
To me this is translating in three different entries in the changelog, something like:

```
### Added
- Add `Serializable` trait to all structures

### Removed
- Remove manual implementation of `from_bytes` and `to_bytes` from all structures

### Changed
- Change return value of `from_bytes` from  `Option` / `CtOption` into `Result<Self, Error>`
```

Something similar to that.

Notice also that where usually the verb of action itself is present ("Change", "Add", etc) the section is using the past tense ("Changed", "Added", etc).

But this is more about the convention, we can postpone the refactor of the changelog for later. But I would rephrase the change in a better way.

If you want a generic reference, you can use: https://keepachangelog.com/en/1.0.0/